Sources/DoubleExtensions.swift

```diff
@@ -14,6 +14,9 @@ extension Double {
 
     /// EZSE: Converts Double to Int
     public var toInt: Int { return Int(self) }
+    
+    /// EZSE: Converts Double to CGFloat
+    public var toCGFloat: CGFloat { return CGFloat(self) }
```

Comment:
I guess you can define a similar if def strategy in your unit tests, and then run the worst case values. (DBL MIN, MAX and FLT MIN, MAX). 

But then again I dont remember which device type is default in the build target. Our build targets are only based on platform type (ios vs tvos).